doc/user/data/sql_funcs.yml

```
@@ -471,7 +471,22 @@
     description: Convert `s` to uppercase.
 
   - signature: 'reverse(s: str) -> str'
-    description: Reverse the characters in `s`.
+    description: |
+      Reverse the characters in `s`.
+
+      Splits the string at occurrences of delimiter and returns a text array of
+      the split segments. 
+      
+      If `delimiter` is NULL, each character in the string will become a
+      separate element in the array.
+      
+      If `delimiter` is an empty string, then the string is treated as a single
+      field. 
+      
+      If `null_string` is supplied and is not NULL, fields matching that string
+      are replaced by NULL.
+
+      For example: `string_to_array('xx~~yy~~zz', '~~', 'yy')` → `{xx,NULL,zz}`
```

Heh ... sorry for the whitespaces 😄 
```suggestion
    description: |
      Reverse the characters in `s`.

      Splits the string at occurrences of delimiter and returns a text array of
      the split segments.

      If `delimiter` is NULL, each character in the string will become a
      separate element in the array.

      If `delimiter` is an empty string, then the string is treated as a single
      field.

      If `null_string` is supplied and is not NULL, fields matching that string
      are replaced by NULL.

      For example: `string_to_array('xx~~yy~~zz', '~~', 'yy')` → `{xx,NULL,zz}`
```